modules/core/src/main/java/org/apache/ignite/internal/thread/ThreadOperation.java

```diff
@@ -25,8 +25,8 @@ public enum ThreadOperation {
     STORAGE_READ,
     /** Storage write. */
     STORAGE_WRITE,
-    /** Take replication protocol snapshot meta (used when initiating a full state transfer of a partition). */
-    TAKE_SNAPSHOT_META,
+    /** Read TX State storage. */
+    TX_METASTORAGE_READ,
```

Comment:
TX_METASTORAGE isn't a canonical name, I believe it'll be confused with common meta storage. TX_STATE_STRORAGE fits better from my point of view.
Anyway, do re really need special operation to read from tx state storage? If we do we probably should also have TX_STATE_STORAGE_WRITE. All in all, I believe that from the treads perspective it doesn't really matter which storage it is, it's just an ability to perform long running storage related actions.